.eslintrc.js

```diff
@@ -39,6 +39,8 @@ module.exports = {
       'error',
       'always'
     ],
+    // suppress errors for missing 'import React' in files
+    'react/react-in-jsx-scope': 'off',
```

Comment:
This allows us to remove `import React` from the top of each file, since it's no longer needed since 2020: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html